content/reference/routing.md

```diff
@@ -0,0 +1,406 @@
+# Routing
+Choo is built up out of two parts: stores and views. In order to render a view,
+it must be added to the application through `app.route()`. This is the router.
+
+In many other frameworks, routing is done by a separate library. We found that
+routing is common in most applications, so making it part of the framework
+makes sense.
+
+Routing has a few parts to it. Routing must handle the browser's history API
+(e.g.  going forward & backwards). It must handle anchor tags, and programmatic
+actions. There's more than a few moving pieces.
+
+To perform routing, Choo uses a [Trie](https://en.wikipedia.org/wiki/Trie) data
+structure. This means that our routings is fast, and the order in which routes
+are added doesn't matter.
+
+_Note: It's recommended to read the [views]('/reference/views) chapter first, as
+we'll assume that you're already familiar with how views work. This chapter is
+intended to give an overview of how routing works in Choo._
+
+## Static routing
+Every application needs an entry point. Routes in Choo are defined relative to
+the host. The route `'/'` maps to `www.mysite.com`. The route `'/foo'` maps to
+`www.mysite.com/foo`.
+
+```js
+var html = require('choo/html')
+var choo = require('choo')
+
+var app = choo()                   // 1.
+app.route('/', view)               // 2.
+app.mount('body')                  // 3.
+
+function view () {                 // 4.
+  return html`
+    <body>Hello World</body>
+  `
+}
+```
+
+1. We need an instance of Choo to add our routes to, so let's create that
+   first.
+2. We're going to add a view on the `'/` route. This means that if people
+   navigate to `oursite.com`, this will be the route that is enabled.
+3. Now that we have our view, we can start rendering our application.
+4. We declare our view at the bottom of the page. Thanks to [scope
+   hoisting](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting), we
+   can use it higher in the code. For now it doesn't really matter what's in
```

Comment:
`higher in the code` can be replaced with `it doesn't matter where in the code we use it`